src/javascript/app/pages/trade/price.js

```diff
@@ -195,7 +195,18 @@ const Price = (() => {
         const currency            = CommonFunctions.getVisibleElement('currency');
 
         if (!h4) return;
-        const display_text = type && contract_type ? contract_type[type] : '';
+
+        const form_name = sessionStorage.getItem('formname');
```

Comment:
Doing checks by just using sessionStorage or localStorage is not a reliable way of writing your code IMO.
Can you get these values from the store or somewhere else ?

Reply:
Looked into this. Based on what I can tell from the repo it looks like session storage parameters are extracted and set from query parameters. A bit confused as though how query parameters are initially set though. Please advise.